src/components/Parallax.tsx

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from 'preact/hooks'
+import ChildrenProp from 'models/ChildrenProp'
+
+export default function ({
+  maxYPixelsDisplacedFromOrigin,
+  children,
+}: ChildrenProp & { maxYPixelsDisplacedFromOrigin: number }) {
+  const [offsetY, setOffsetY] = useState(0)
+  const handleScroll = () => setOffsetY(window.pageYOffset)
+
+  useEffect(() => {
+    window.addEventListener('scroll', handleScroll)
+
+    return () => window.removeEventListener('scroll', handleScroll)
+  }, [])
+
+  return (
+    <div
+      style={{
+        transform: `translateY(-${
+          (offsetY * maxYPixelsDisplacedFromOrigin) / 500
```

Comment:
Why 500 here?

Reply:
Just played around with different values to divide by until the asset moved maxYPixelsDisplacedFromOrigin pixels. In the new commit, it's 8000 because 16px = 1rem